Storage/storage.h

```diff
@@ -73,6 +73,26 @@ namespace storage {
 // 2) Splits the type into `T_PERSISTED_EVENT_1` and `T_PERSISTED_EVENT_2` to enable persisting deletions too.
 
 // clang-format off
+#define CURRENT_STORAGE_FIELD_EXCLUDE_FROM(feature, entry_name)                                           \
+  template<> struct current::storage::FieldParticipatesIn##feature<entry_name::T_FIELD_TYPE<              \
+                                                                   entry_name::T_ENTRY,                   \
+                                                                   entry_name::T_PERSISTED_EVENT_1,       \
+                                                                   entry_name::T_PERSISTED_EVENT_2 >> {   \
+    static constexpr bool participates = false;                                                           \
+  }
+
+#define CURRENT_STORAGE_DEFINE_FEATURE(feature) \
+  template <typename T>                         \
+  struct FieldParticipatesIn##feature {         \
+    static constexpr bool participates = true;  \
+  }
+
+// defined here because Storage/api.h doesn't need to be included at the point where the EXCLUDE_FROM is used
+CURRENT_STORAGE_DEFINE_FEATURE(RESTfulAPI);
+
+#define CURRENT_STORAGE_FIELD_EXCLUDE_FROM_RESTFUL_API(entry_name)   \
+			CURRENT_STORAGE_FIELD_EXCLUDE_FROM(RESTfulAPI, entry_name)
+
```

Comment:
Hi @nyospe ,

I've added tests and tweaked the logic slightly:
* https://github.com/dkorolev/Current/commit/0d1b1fd9bfcced55de5ec2037e8e3fa990a1b450
* https://github.com/dkorolev/Current/commit/7311df419315737b79159cf740f69c87c69bd313

It's going to master as @mzhurovich would be merging https://github.com/c5t/Current/pull/373.

Thanks!
Dima